packages/react-query/src/__tests__/useInfiniteQuery.test.tsx

```diff
@@ -81,7 +81,7 @@ describe('useInfiniteQuery', () => {
 
     renderWithClient(queryClient, <Page />)
 
-    await vi.advanceTimersByTimeAsync(100)
+    await vi.advanceTimersByTimeAsync(0)
```

Comment:
```suggestion
    await vi.advanceTimersByTimeAsync(10)
```

Could you add queryFn like below

```tsx
    function Page() {
      const state = useInfiniteQuery({
        queryKey: key,
        queryFn: ({ pageParam }) => sleep(10).then(() => Number(pageParam)),
        getNextPageParam: (lastPage) => lastPage + 1,
        initialPageParam: 0,
      })
      states.push(state)
      return null
    }
```

Reply:
@manudeli 

I reflected your opinion. 👍